cli/loader_lib.c

```diff
@@ -345,6 +345,8 @@ static char *libstdcxxprobe(void)
             free(path);
             return NULL;
         }
+        // Ensure that `path` is zero-terminated.
+        path[pathlen] = '\0';
```

Comment:
Are we sure it has room for the extra byte?

Reply:
yes: https://github.com/JuliaLang/julia/blob/71115975bebfc82952309712a465e665527787d3/cli/loader_lib.c#L220